include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/RoborazziOptions.kt

```diff
@@ -38,6 +39,40 @@ fun roborazziDefaultResizeScale(): Double {
   return checkNotNull(System.getProperty("roborazzi.record.resizeScale", "1.0")).toDouble()
 }
 
+@ExperimentalRoborazziApi
+sealed interface RoborazziRecordFilePathStrategy {
+  val propertyValue: String
+
+  object RelativePathFromCurrentDirectory : RoborazziRecordFilePathStrategy {
+    override val propertyValue: String
+      get() = "RelativePathFromCurrentDirectory"
+  }
+
+  object RelativePathFromRoborazziContextOutputDirectory : RoborazziRecordFilePathStrategy {
+    override val propertyValue: String
+      get() = "RelativePathFromRoborazziContextOutputDirectory"
+  }
+}
+
+
+@ExperimentalRoborazziApi
+fun roborazziRecordFilePathStrategy(): RoborazziRecordFilePathStrategy {
+  return when (
+    System.getProperty(
+      "roborazzi.record.filePathStrategy",
```

Comment:
The property name has been changed.
https://github.com/takahirom/roborazzi/pull/172#discussion_r1339384610